plotly/tests/test_core/test_tools/test_figure_factory.py

```diff
@@ -1125,6 +1125,245 @@ def test_table_with_index(self):
                                                 'zeroline': False}}}
         self.assertEqual(index_table, exp_index_table)
 
+
+class TestGantt(TestCase):
+
+    def test_validate_gantt(self):
+
+        # validate the basic gantt inputs
+
+        df = [dict(Task='Job A',
+                   Start='2009-02-01',
+                   Finish='2009-08-30',
+                   Complete='a')]
+
+        pattern2 = ("In order to use an indexing column and assign colors to "
+                    "the values of the index, you must choose an actual "
+                    "column name in the dataframe or key if a list of "
+                    "dictionaries is being used.")
```

Comment:
🐄 if you're not using the `'`, character, we typically try to use `'` instead of `"` for the enclosing string literal notation.